### ISSUE TITLE
Public Works content audit

### ISSUE THREAD

**Issue by LauraTrujillo** [cityofaustin/techstack#2548]
home page: https://www.austintexas.gov/department/public-works

**Comment by mark-swanson** [comment]
Ready to go! https://drive.google.com/drive/folders/19l_uFP3vIh7na1uwm9N8HXzySosGI6HP

**Comment by LauraTrujillo** [comment]
@mark-swanson can you keep diving into the program pages?

**Comment by mark-swanson** [comment]
Yeah, that kind of what Capital Contracting is. Maybe combine into one doc?

**Comment by desigonz** [comment]
assigned to @mark-swanson to review

**Comment by annieszot** [comment]
Reviewed 9/4/2019
Program pages need to be audited/indexed. Although capital contracts do support Public Works programs, should keep program pages as part of Public Works audit (see comments above) 
Proposed next steps are: 
- [ ] Complete audit for program pages
- [ ] Make issues for Public Works content transition and backlog

**Comment by desigonz** [comment]
Thanks @annieszot! I've unassigned Mark and put this back "on deck" for the content audit team.

**Comment by matthewbarron** [comment]
Estimate of 3 sounds right

**Comment by Jo-Dwyer** [comment]
[current inventory](https://docs.google.com/spreadsheets/d/1c_GKUshAWAh18xcpHszMCvJI141lricOsIuQbgoyqIg/edit#gid=0) @matthewbarron is this completed?

**Comment by matthewbarron** [comment]
Did a quick look into it and can say it sure doesn’t look completed – I think Mark did this one a while back. I’ll update the sheet to say it needs to be revisited asap.
